test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,7 +25,7 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 38,101 IPv4 -statSemWait
+brokerStart CB 38,101 IPv4 -statSemWait --cache
```

Comment:
We have a nice little mess between '--' and '-' options for the long names ...

-statSemWait is actually *not* an option for brokerStart.
brokerStart just copies it down to the broker.
--cache *is* an option for brokerStart, and just like all other script functions (I hope), it uses '--'.

So, do you still want to change '--cache' ?

The best option actually would be to modify the broker to use '--' instead of '-' for the long options. Support both '-' and '--', to finally drop '-'.

And, of course, the entire arg-parsing for brokerStart should be modified. That is in a separate issue, I believe.